src/psyclone/tests/psyGen_test.py

```diff
@@ -45,6 +45,7 @@
 # PSyFactory, TransInfo, Transformation
 import os
 import sys
+import logging as pylogging
```

Comment:
Is the logging in `from fparser import api as fpapi, logging` a transitive import? Can we not simply use `import logging`? (May not be necessary here if we go for a DeprecationWarning)

Reply:
I think so, I've removed the "pylogging" and logging import from api so this file just imports logging as normal now.